app/routes.php

```diff
@@ -62,6 +64,13 @@ function (RouteCollectorProxy $paymentMethodsGroup) {
     $app->post('/hooks/stripe', Hooks\StripePaymentsUpdate::class);
     $app->post('/hooks/stripe-connect', Hooks\StripePayoutUpdate::class);
 
+    // Requests from Salesforce
+
+    $app->post(
+        '/charities/{salesforceId:[a-zA-Z0-9]{18}}/update-required',
```

Comment:
```suggestion
        '/hooks/charities/{salesforceId:[a-zA-Z0-9]{18}}/update-required',
```

Might be worth having this prefix consistent to help spot everything inbound at a glance?

Reply:
Not sure if this is really a webhook, I think a defining feature of a webhook is that the address is user-configurable on the sending application side. But it doesn't really matter much and in SF the line between user and developer is pretty blurry. Done in cb28ca10